### PR TITLE
[Merged by Bors] - feat: port Topology.Sheaves.Stalks

### PR DESCRIPTION
**Pull request by yuma-mizuno**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by yuma-mizuno** [comment]
We cannot put the ext attribute to `germ_ext`.

**Comment by jjaassoonn** [comment]
> We cannot put the ext attribute to `germ_ext`.

I don't think this is a problem because other similar file I have ported `ext` behaves quite differently from its Lean3 counterpart, so if `germ_ext` is needed, we can just use `refine germ_ext ?_ ?_ ...`

File: Mathlib/Topology/Sheaves/Stalks.lean [hunk 1]
```diff
@@ -130,51 +138,54 @@ stalk of `f _ * F` at `f x` and the stalk of `F` at `x`.
 -/
 def stalkPushforward (f : X ⟶ Y) (F : X.Presheaf C) (x : X) : (f _* F).stalk (f x) ⟶ F.stalk x := by
   -- This is a hack; Lean doesn't like to elaborate the term written directly.
-  trans
-  swap
-  exact colimit.pre _ (open_nhds.map f x).op
-  exact colim.map (whisker_right (nat_trans.op (open_nhds.inclusion_map_iso f x).inv) F)
+  -- swap
+  -- Porting note: In the above proof, `trans` do nothing.
```
Comment (jcommelin) [inline Mathlib/Topology/Sheaves/Stalks.lean:142]:
```suggestion
  -- Porting note: The original proof was `trans; swap`, but `trans` does nothing.
```

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**